keps/sig-api-machinery/555-server-side-apply/README.md

```diff
@@ -286,6 +287,67 @@ The conversion between the two and creating the diff was complex and would have
 
 - 12/2019 [#86083](https://github.com/kubernetes/kubernetes/pull/86083) implementing a poc for the described approach
 
+
+#### API Audit
+
+The `ManagedFields` fields of an object in the API audit log may not be very useful. We want to provide a mechanism, 
+so the cluster operator can opt in so that the managed fields can be omitted from the audit log.
+
+We propose the following changes to the `audit.k8s.io/Policy` API that provides the cluster operator with a more 
+granular way to control the omission of managed fields in audit log:
+```go
+type Policy struct {
+	// +optional
+	OmitManagedFields bool `json:"omitManagedFields,omitempty"`
+}
+
+type PolicyRule struct {
+	// +optional
+	OmitManagedFields *bool `json:"omitManagedFields,omitempty"`
+}
+```
+The above API changes will be introduced in `v1`, `v1beta1` and `v1alpha1` of `audit.k8s.io`
```

Comment:
we never roundtrip these, do we? So technically we could do this in v1 only.

Reply:
But there's no harm in doing it in all, no?

Reply:
only that we encourage use of v1 by not implementing new stuff in old versions.